test/setToken-base.spec.ts

```diff
@@ -0,0 +1,355 @@
+import * as chai from "chai";
+const { expect, assert } = chai;
+
+import { BigNumber } from "bignumber.js";
+
+// Types
+import { Address, UInt } from "../types/common.js";
+
+// Contract types
+// import { StandardTokenMockContract } from "../types/generated/standard_token_mock";
+// import { SetTokenContract } from "../types/generated/set_token";
+
+// Artifacts
+const SetToken = artifacts.require("SetToken");
+const StandardTokenMock = artifacts.require("StandardTokenMock");
+
+// Testing Set up
+import { BigNumberSetup } from "./config/bignumber_setup";
```

Comment:
we can move this into some `jest` configuration: https://facebook.github.io/jest/docs/en/configuration.html#setupfiles-array

Reply:
nvm this is using truffle/mocha...i wonder if this file structure would work to pull this out to a common place https://stackoverflow.com/questions/24153261/joining-tests-from-multiple-files-with-mocha-js/26791892#26791892